File: php/PROBLEMS/1101.php
<?php

function funcao($x, $y){
    $sum = 0;
    for ($i=$y;$i<=$x;$i++){
        printf("%d ", $i);
        $sum+=$i;
    }
    printf("Sum=%d\n", $sum);
}

while (true){
    $e = fgets(STDIN);
    $e = explode(' ', $e);
    $x = intval($e[0]);
    $y = intval($e[1]);
    if ($y > $x){
        $aux = $y;
        $y = $x;
        $x = $aux;
    }
    if ($x <= 0 || $y <= 0){
        return;
    }
    funcao($x, $y);
}

?>

File: c/PROBLEMS/1101.c
#include <stdio.h>

void funcao(int x, int y)
{
    int sum = 0, i;
    for (i = y; i <= x; i++)
    {
        printf("%d ", i);
        sum += i;
    }
    printf("Sum=%d\n", sum);
}

int main()
{
    while (1)
    {
        int x, y, aux;
        scanf("%d %d", &x, &y);
        if (y > x)
        {
            aux = y;
            y = x;
            x = aux;
        }
        if (x <= 0 || y <= 0)
        {
            break;
        }
        funcao(x, y);
    }
}
